### PR TITLE
refactor: replace python_model with propensity model

### PR DESCRIPTION
**Pull request by shekhar-rudder**
# Description

Replace with adequate description for this PR as per [Pull Request document](https://www.notion.so/rudderstacks/RS360-PR-Conventions-and-Release-Process-Standardisation-833d3666bbfd4e3d93a27475cc42b435)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Developer Checklist

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] Lint rules pass locally
- [ ] If applicable, are your project model changes backward compatible. Note that inputs should not be removed or renamed as it will break existing projects.

## Code Reviewer Checklist

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Migration to the latest schema, if required
- [ ] If applicable, are your project model changes backward compatible. Note that inputs should not be removed or renamed as it will break existing projects.

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: pb_project.yaml
Comment (shekhar-rudder) [inline pb_project.yaml:37]: Release this package before merging the PR

File: models/profiles-ml.yaml
Comment (shekhar-rudder) [inline models/profiles-ml.yaml:21]: @dpatchigolla Do we want to cut down the number of inputs for the model ? Unlike the feature table model as in input, if an entity var is disabled, it will disable the propensity model.

Comment (dpatchigolla) [reply models/profiles-ml.yaml:21]: Yes. Let's bring this down to following:
```
        - days_since_last_seen
        - days_since_account_creation
        - active_days_in_past_7_days
        - active_days_in_past_365_days
        - total_sessions_till_date
        - total_sessions_last_week
        - total_sessions_90_days
        - total_sessions_365_days
        - avg_session_length_in_sec_overall
        - avg_session_length_in_sec_last_week
        - avg_session_length_in_sec_365_days
        - is_churned_7_days
        - is_churned_30_days
        - is_churned_90_days
```

This skips the categorical and array type features, which are typically more noisy. The above features all rely only on pages/tracks and shouldn't usually cause errors either.

File: models/profiles-ml.yaml
Comment (dpatchigolla) [inline models/profiles-ml.yaml:15]: Let's add is_feature: True for the percentile columns? That's default, but for a end user this helps it clearer esp. when the score column has this set to False.

File: models/profiles-ml.yaml
Comment (shekhar-rudder) [reply models/profiles-ml.yaml:21]: Updated

**Review by dpatchigolla — APPROVED** [review]
I realise I left a comment earlier but didn't post. Approving with that comment.

File: models/profiles-ml.yaml
Comment (shekhar-rudder) [reply models/profiles-ml.yaml:15]: Added

**Pull request merged**
